packages/flutter/lib/src/painting/strut_style.dart

```diff
@@ -33,16 +34,22 @@ import 'text_style.dart';
 ///
 /// The sum of these four values is the total height of the line.
 ///
-/// The `ascent + descent` is equivalent to the [fontSize]. Ascent is the font's
-/// spacing above the baseline without leading and descent is the spacing below the
-/// baseline without leading. Leading is split evenly between the top and bottom.
-/// The values for `ascent` and `descent` are provided by the font named by
-/// [fontFamily]. If no [fontFamily] or [fontFamilyFallback] is provided, then the
-/// platform's default family will be used.
+/// Ascent is the font's spacing above the baseline without leading and
+/// descent is the spacing below the baseline without leading. Leading is
+/// split evenly betweenthe top and bottom. The values for `ascent` and
+/// `descent` are provided by the font named by [fontFamily]. If no
```

Comment:
Am I missing a diagram somewhere?  Or even just a wikipedia link that has one?

Reply:
Making one :)

Reply:
Diagram is at https://github.com/flutter/assets-for-api-docs/pull/86

Reply:
Will update the docs to reference it once it lands.

Reply:
https://github.com/flutter/assets-for-api-docs/pull/87